docroot/custom_jsps/html/portlet/login/login.jsp

```diff
@@ -0,0 +1,183 @@
+			<liferay-ui:error exception="<%= CookieNotSupportedException.class %>" message="authentication-failed-please-enable-browser-cookies" />
+			<liferay-ui:error exception="<%= NoSuchUserException.class %>" message="authentication-failed" />
+			<liferay-ui:error exception="<%= PasswordExpiredException.class %>" message="your-password-has-expired" />
+			<liferay-ui:error exception="<%= UserEmailAddressException.class %>" message="authentication-failed" />
+			<liferay-ui:error exception="<%= UserLockoutException.class %>" message="this-account-has-been-locked" />
+			<liferay-ui:error exception="<%= UserPasswordException.class %>" message="authentication-failed" />
+			<liferay-ui:error exception="<%= UserScreenNameException.class %>" message="authentication-failed" />
+			<h4>Welcome to Liferay courses</h4>
+			<aui:fieldset>
+
+				<%
+				String loginLabel = null;
+
+				if (authType.equals(CompanyConstants.AUTH_TYPE_EA)) {
+					loginLabel = "email-address";
+				}
+				else if (authType.equals(CompanyConstants.AUTH_TYPE_SN)) {
+					loginLabel = "screen-name";
+				}
+				else if (authType.equals(CompanyConstants.AUTH_TYPE_ID)) {
+					loginLabel = "id";
+				}
+				%>
+
+				<aui:input autoFocus="<%= windowState.equals(LiferayWindowState.EXCLUSIVE) || windowState.equals(WindowState.MAXIMIZED) %>" cssClass="clearable" label="<%= loginLabel %>" name="login" showRequiredLabel="<%= false %>" type="text" value="<%= login %>">
+					<aui:validator name="required" />
+				</aui:input>
+
+				<aui:input name="password" showRequiredLabel="<%= false %>" type="password" value="<%= password %>">
+					<aui:validator name="required" />
+				</aui:input>
+
+				<span id="<portlet:namespace />passwordCapsLockSpan" style="display: none;"><liferay-ui:message key="caps-lock-is-on" /></span>
+
+				<!--
+				<c:if test="<%= company.isAutoLogin() && !PropsValues.SESSION_DISABLED %>">
+					<aui:input checked="<%= rememberMe %>" name="rememberMe" type="checkbox" />
+				</c:if> 
+				-->
+			</aui:fieldset>
+
+			<aui:button-row>
+				<aui:button type="submit" value="sign-in" />
+			</aui:button-row>
+		</aui:form>
+
+		<!-- <liferay-util:include page="/html/portlet/login/navigation.jsp" /> -->
+
+		<aui:script use="aui-base">
+			var form = A.one(document.<portlet:namespace />fm);
+
+			form.on(
+				'submit',
+				function(event) {
+					var redirect = form.one('#<portlet:namespace />redirect');
+
+					if (redirect) {
+						var redirectVal = redirect.val();
+
+						redirect.val(redirectVal + window.location.hash);
+					}
+
+					submitForm(form);
+				}
+			);
+
+			var password = form.one('#<portlet:namespace />password');
+
+			if (password) {
+				password.on(
+					'keypress',
+					function(event) {
+						Liferay.Util.showCapsLock(event, '<portlet:namespace />passwordCapsLockSpan');
+					}
+				);
+			}
+		</aui:script>
+	</c:otherwise>
+</c:choose>
```

Comment:
Try to avoid <% and <%=. We can create variables from the controller then pass them as Request's Attribute to use with ${} in jsp. This belongs to the best practices. However, at this time, this requirement is not important.